randomMenuLunch.rb

```diff
@@ -0,0 +1,29 @@
+# Random Menu Generator (RMG Lunch)
+
+descriptor1 = ["fresh", "bite-size", "aromatic", "aged", "famous", "classic", "delicious", "delightful", "fantastic", "traditional"]
+descriptor2 = ["baked", "boiled", "caramelized", "creamy", "crunchy", "rosted", "sliced", "smoked", "grilled", "crisp"]
+ingredient = ["eggs", "cheese", "chicken", "tomatoes", "steak", "salmon", "beef", "garlic", "mushrooms", "sausage"]
+
+(1..ingredient.length).each do |i|
+  a = rand(descriptor1.length)
+  b = rand(descriptor2.length)
+  c = rand(ingredient.length)
```

Comment:
I like how your random word from each array takes into account how many items are in each array, since it essentially lets you accommodate arrays of any length.

It'd be nice to use more descriptive variable names than `a`, `b`, and `c`, but I think more verbose variable names might be overkill for this project.